python/test/test_tracking.py

```
@@ -126,6 +126,67 @@ def test_tracking(self):
                             #     if gravity is not None:
                             #         self.assertAlmostEqual(gravity[1], 9.81, msg=f"iteration {i}")
 
+    def _create_multisensor_tracker(self, rig, config):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_create_multisensor_tracker`

(ANN202)